src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -1,48 +1,48 @@
-params:
+–params:
 - name: text
   type: string
   required: true
-  description: If `html` is set, this is not required. Text to use within the notification banner. If `html` is provided, the `text` argument will be ignored.
+  description: The text that displays in the notification banner. If you set `html`, this is not required and the `text` option is ignored.
 - name: html
   type: string
   required: true
-  description: If `text` is set, this is not required. HTML to use within the notification banner. If `html` is provided, the `text` argument will be ignored.
+  description: The HTML to use within the notification banner. If you set `text`, this is not required. If you set `html`, the `text` option is ignored.
 - name: title
   type: string
   required: false
-  description: Title text to use within the notification banner. Defaults to 'Important' ('Success' for success type and 'Error' for error type). If `titleHtml` is supplied, the `title` argument will be ignored.
+  description: The title text that displays in the notification banner. If you do not set `type`, the `title` text defaults to 'Important'. If you set `type` to `success`, the `title` text defaults to 'Success'. If you set `type` to `error`, the `title` text defaults to 'Error'. If you set `titleHtml`, the `title` option is ignored.
 - name: titleHtml
   type: string
   required: false
-  description: Title HTML to use within the notification banner. If `titleHtml` is provided, the `title` argument will be ignored.
+  description: The title HTML to use within the notification banner. If you set `titleHtml`, the `title` option is ignored.
 - name: titleHeadingLevel
   type: string
   required: false
-  description: Heading level, from 1 to 6. Default is `2`.
+  description: Sets heading level. The minimum is `1`. The maximum is `6`. The default is `2`.
```

Comment:
Is it worth clarifying that it sets the heading level for the __title only__? I guess the option name tells you that, so maybe it's not needed 🤷

Reply:
Good suggestion. No harm clarifying this (imo)!

Reply:
```suggestion
  description: Sets heading level for the title only. The minimum is `1`. The maximum is `6`. The default is `2`.
```